LeanKit-Labs/tag-release
# Get tags from repo for version bump using --qa feature instead of manual entry

### Short description of the work completed

> No longer have user enter version of tag they want to update the dependency to, now pulls all tags from the repo they are trying to update and prompts them to select the version they wish to use.

### Steps to test (if not obvious)

1. Follow flow using `--qa` flag on command line for tag-release
2. Should see a prompt of tags for each dependency version you want to update instead of a manual entry.

### For Reviewer Use Only

  patch)
